Doc/library/sqlite3.rst

```diff
@@ -1201,6 +1233,30 @@ Connection objects
    .. _SQLite limit category: https://www.sqlite.org/c3ref/c_limit_attached.html
 
 
+   .. method:: getconfig(op, /)
+
+      Query a boolean connection configuration option.
+
+      :param int op:
+         A :ref:`SQLITE_DBCONFIG code <sqlite3-dbconfig-constants>`.
+
+      :rtype: bool
+
+      .. versionadded:: 3.12
+
+   .. method:: setconfig(op, enable=True, /)
+
+      Set a boolean connection configuration option.
```

Comment:
```suggestion
      Set a connection configuration option.
      Currently, only boolean configuration options are supported.
      In the future, other types of options may be supported.
```

Reply:
Hmm, I'm not sure the "We might do this in the future, possibly" wording is useful. Users can't rely on it _maybe_ changing in the future. I think I'd just do something like this:

```suggestion
      Query the value of the connection configuration option *op*.
      Only boolean configuration options are supported.
```

Reply:
Same as https://github.com/python/cpython/pull/103506/files#r1175758343

```suggestion
      Set the connection configuration option *op*.
      Only boolean configuration options are supported.
```

Reply:
Since, we already are using parameter lists, perhaps the current wording is good enough: _Set a boolean connection configuration option_?

Reply:
Let us keep it as it is in the PR.